### ISSUE TITLE
[kaeld on 2010-03-22 10:13:50] : iDOM Support

### ISSUE THREAD

**Issue by dglo** [dglo/pdaq#1298]
No associated GitHub commit

**Comment by dglo** [comment]
[kaeld on 2010-05-11 18:33:27]
From: Thorsten Stezelberger <TStezelberger@lbl.gov>
Date: May 11, 2010 11:21:38 AM CDT
To: John Jacobsen <john@mail.npxdesigns.com>
Cc: Ryan Bay <bay@berkeley.edu>, hagar@icecube.wisc.edu
Subject: FPGA update

Hi John,

I just checked a new domapp FPGA into the CVS branch and I updated the API document in docuhide. This update is for the ATWD acquisition counter Hagar requested and the Inclinometer code for Ryan. The 4xx branch is tagged build539-4xx.

Thorsten
